inginious/frontend/templates/course_admin/settings.html

```diff
@@ -340,6 +367,8 @@ <h2>{{_("Course settings")}}</h2>
         updateLTI();
         $("#lti").click(updateLTI);
         $("#lti").change(updateLTI);
+        hideAcType();
+        $("[name='registration_ac']").change(hideAcType);
```

Comment:
I think you can add an id to avoid a complex name-based selector.

Reply:
The id is defined only once here
https://github.com/UCL-INGI/INGInious/blob/2b9ea8d0e38f9e6750f4b365612fd56209bf7c11/inginious/frontend/templates/course_admin/settings.html#L262
and the next radios use the same name to reference this id.
Matching the id is not sufficient to detect a change in the radio value (it detects when the radio is clicked but not when another radio is selected).